tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_gcs_using_spark.py

```diff
@@ -1,5 +1,5 @@
 """
-To run this code as a local test, use the following console command:
+To run this code as a local environment, use the following console command:
```

Comment:
This change ensures that the `docs-snippets` stage is run. It will be removed before merging.